### PR TITLE
Add logic to recognize missing images

### PR DESCRIPTION
**Pull request by sedrubal**

### CODE REVIEW

File: kastenwesen.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 Actions explained:
   status: show status
-  rebuild: rebuild and restart. Takes care of dependencies.
```
Comment (mgmax) [inline kastenwesen.py:40]: some linebreaks would be nice here

File: kastenwesen.py [hunk 2]
```diff
@@ -313,6 +324,17 @@ def rebuild(self, ignore_cache=False):
...
 
+    @property
+    def is_built(self):
+        """Return True if image for this container exists locally."""
+        return any(
+            any(
+                tag == NAMESPACE + self.name + (':latest' if ':' not in self.name else '')
+                for tag in image['RepoTags']
+            )
+            for image in DOCKER_API_CLIENT.images(name=NAMESPACE + self.name)
```
Comment (mgmax) [inline kastenwesen.py]: I think this docker-specific implementation actually belongs to `DockerContainer`, because `AbstractContainer` is generic.

**Review by mgmax — DISMISSED** [review]
Looks good except for two minor things - see comments. I assume you tested the code, as I don't have time for that.

**Comment by sedrubal** [comment]
@mgmax Thanks for your review. I tested it so far that I think we can merge it.

**Pull request merged**